specs/functionality.md

```diff
@@ -1,115 +1,129 @@
-            * Recognize entry list from a photo (optional)
-                * Upload photo
-                * Send photo to external service for analysis
+            * [1] Add entries to the list
+                * [3] Add price
+                * [2] Tag as vegetarian
+                * [3] Add different tags (vegan, halaal, locally sourced etc.) (optional)
+                    * [4] Support user-created tags
+            * Upload menu photo
+                * [3] Recognize entry list from a photo (optional)
         * Browse menus
-            * Display options with ratings
+            * [1] Display options with ratings
                 * Stars (integers in range [1..5])
                 * Text reviews (optional)
-                * Photos of dishes (optional)
-                * Prices of dishes (optional)
-            * Display photo of menu board if uploaded and no text options (optional)
-        * Update menus
-        * Delete menus
-        * Post menu to Facebook (optional)
-        * Export menu as PDF (optional)
+                * [4] Photos of dishes (optional)
+                * [3] Prices of dishes (optional)
+            * [3] Display photo of menu board if uploaded and no text options (optional)
+        * [2] Update menus
+        * [1] Delete menus
+        * [4] Post menu to Facebook (optional)
+        * [4] Export menu as PDF (optional)
     * Manage cameras
-        * Register new camera and generate authentication key
-        * List cameras
+        * [1] Register new camera and generate authentication key
+        * [2] List cameras
             * Show state (on/off/lost connection)
-        * Update camera
+        * [2] Update camera
             * Reset auth key
-        * Unregister camera
+        * [1] Unregister camera
     * Browse analytics
-        * Plot cafeteria occupancy
+        * [1] Plot cafeteria occupancy
             * Break by year/month/week/day/hour
-        * Plot review count
+        * [1] Plot review count
             * Break by year/month/week/day/hour
-        * Plot different charts together for correlation analysis
-        * Popularity of dishes based on canteens owners data (optional)
+        * [2] Plot different charts together for correlation analysis
+        * [4] Popularity of dishes based on canteens owners data (optional)
 5. Client app
     * Browse canteens
         * View canteen
-            * Open/closed
-            * Live number of visitors (relatively to the full capacity)
-                * Create widget for mobile devices (optional)
-            * Display predicted occupance (optional)
-            * Has specified food - based on tags (optional)
-            * List menu options (see below)
-            * List historical menus (see below)
-        * Filter canteen list
-            * Open/closed
-            * Minimum rating
-            * Has tags (optional)
-        * Find nearby canteens based on used location (optional on Android)
+            * [2] Open/closed
```

Comment:
based on hour in phone and the opening hours from owner so can be [1]

Reply:
Opening hours are priority 3 right now. I'll make both 2.